tests/test_cmd_base.py

```diff
@@ -207,6 +208,23 @@ def test_execute(self, depfile_name):
             '--mine', 'min'])
 
 
+    @mock.patch('doit.cmd_base.Globals')
+    def test_execute_provides_dep_manager(self, mock_globals, depfile_name):
+        mock_globals.dep_manager = None
+        members = {'task_xxx1': lambda: {'actions': []}}
+
+        class MockTaskLoader(ModuleTaskLoader):
+            def load_tasks(self, cmd, pos_args):
+                assert mock_globals.dep_manager, 'dep_manager not set before tasks are loaded'
```

Comment:
I think add an ``assert`` explanation is not a good practice with py.test.
py.test will rewrite the assert and give a better error message when there is no message.
I usually write a comment above the assert when it needs some kind of explanation.

Reply:
I was not aware of [this](https://docs.pytest.org/en/latest/assert.html#assert-details). Thanks for the hint!